sphinx_needs/lsp/esbonio.py

```diff
@@ -0,0 +1,385 @@
+"""Suport Sphinx-Needs language features."""
+import getpass
+import os
+import pathlib
+import re
+from hashlib import blake2b
+from typing import List, Optional, Tuple, Union
+
+from esbonio.lsp import LanguageFeature
+from esbonio.lsp.rst import CompletionContext, DefinitionContext, HoverContext
+from esbonio.lsp.sphinx import SphinxLanguageServer
+from pygls.lsp.types import (
```

Comment:
i don't see the `pygls` dependency in `pyproject.toml`. Where is this getting pulled in?

Reply:
I agree, it should be part of the ``pyproject.toml``, but without any version pinning, as this is the job from the "parent"-dep, in this case ``esbonio``.